feature/gribi/ate_tests/get_rpc_test/get_rpc_test.go

```diff
@@ -317,13 +317,12 @@ func testIPv4LeaderActive(ctx context.Context, t *testing.T, args *testArgs) {
 	// and ensure that only entries for 198.51.100.0/26, 198.51.100.64/26, 198.51.100.128/26
 	// are returned, with no entry returned for 198.51.100.192/64.
 	dc := args.dut.Config()
-	ni := dc.NetworkInstance(*deviations.DefaultNetworkInstance).
-		Protocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, *deviations.StaticProtocolName)
-	static := &telemetry.NetworkInstance_Protocol_Static{
-		Prefix: ygot.String(staticCIDR),
-	}
-	static.GetOrCreateNextHop("0").NextHop = telemetry.UnionString(atePort2.IPv4)
-	ni.Static(staticCIDR).Replace(t, static)
+	ni := dc.NetworkInstance(*deviations.DefaultNetworkInstance).Get(t)
```

Comment:
@robshakir @xw-g  we configure static route in several tests, but they all slightly differ. Should not we just add a util function to unify this? @Hobbydos this can be done as part of this pr, otherwise I can open a separate PR for to do this?